hosting_docs/source/training/0-index.rst

```diff
@@ -0,0 +1,97 @@
+Hosting CommCare: Training Plan
+===============================
+
+Maintaining a stable and up-to-date CommCare HQ environment requires a
+high level of technical skills.
+
+Prerequisite knowledge
+----------------------
+
+Virtualization Stack
+^^^^^^^^^^^^^^^^^^^^
+
+Dimagi recommends that projects use a cloud hosting provider like Amazon
+Web Services, Microsoft Azure, Google Cloud Platform, etc. For some
+projects that is not possible, and those projects host on-premises using
+VMWare, Citrix XenServer, OpenStack or just libvirt, etc.
+
+If the team who will be managing the CommCare environment are also
+responsible for managing the virtual infrastructure, it is important to
+be familiar with the virtualization technology stack.
+
+Online learning resources are available. Cloud hosting providers and
+technology providers  also offer courses.
+
+
+Linux
+^^^^^
+
+CommCare HQ is deployed on the Ubuntu distribution of the GNU/Linux
+operating system. Familiarity with GNU/Linux is an important foundation
+for hosting CommCare HQ. An excellent way to certify that staff have
+that foundation is the LPIC-1 (the
+`Linux Professional Institute <https://www.lpi.org/>`_ Certification
+Level 1).
+
+Course material is available in multiple languages. A lot of course
+material is free of charge at https://learning.lpi.org/. In-person
+training, online training, and exams are offered by affiliated
+organizations in many countries.
+
+
+Course plan
+-----------
+
+The following plan lays out the topics that you need to know as a
+CommCare HQ DevOps engineer.
+
++---------------------------------------------------------+---------+
+| A high-level understanding of the services that make up | 1 hour  |
+| CommCare HQ. See the `CommCare HQ Services Overview`_.  |         |
+|                                                         |         |
++---------------------------------------------------------+---------+
+| A detailed understanding of the installation process:   | 1 hour  |
+| See "Setting up a new CommCare HQ environment           |         |
+| `on a single machine`_" and "Setting up a new CommCare  |         |
+| HQ environment `on a small cluster`_".                  |         |
```

Comment:
I get a 404 when accessing [this](https://dimagi.github.io/commcare-cloud/setup/new_cluster.html) link?

Reply:
Nice catch @Charl1996 . It is [this file](https://github.com/dimagi/commcare-cloud/blob/nh/new_cluster/docs/setup/new_cluster.md) added by (unmerged) PR #5015

Reply:
Out of curiosity...

Since [this](https://github.com/dimagi/commcare-cloud/blob/a20e336e4354bc4e8ea64853c569c5b23d3e2769/docs/setup/new_environment.md) PR touches both monolith and small cluster, does it make sense to use the same link for `on a single machine` and `on a small cluster`?

Also, would it not be..."better"...to link `on a single machine` to the already existing hosting docs [Single Server Production Install](https://commcare-cloud.readthedocs.io/en/latest/installation/monolith/2-install-prod.html)? (Especially because this PR's doc sits in hosting docs)

It seems with [this](https://github.com/dimagi/commcare-cloud/blob/a20e336e4354bc4e8ea64853c569c5b23d3e2769/docs/setup/new_environment.md) PR and [this](https://commcare-cloud.readthedocs.io/en/latest/installation/monolith/2-install-prod.html) docs we're saying the same things in 2 different documents...unless there's a good reason for segregating the different docs?

Reply:
Thank you for the heads-up that I should change this too!

Yeah, the fact that we say exactly the same thing in two places in two different ways is ridiculous. Think of the wasted hours that went into that. Now think of the wasted hours about to be spent fixing it.

Reply:
@Charl1996  f61bd231cae6eb6593eb923fa22e3fb43eafd182

Reply:
(Please excuse the kneejerk response about wasted hours. The new "2-install-prod" doc looks like it's closely based on (a copy of?) the older "new_environment" doc. So not a complete waste of time. Hopefully merging it with the newly merged "new_environment" doc won't be a lot of effort.)